api/docs/release.dox

```diff
@@ -250,6 +250,8 @@ Further non-compatibility-affecting changes include:
  - Added drwrap_get_stats().
  - Added #DRWRAP_NO_DYNAMIC_RETADDRS for reducing drwrap overhead at the cost
    of missing some post-call callbacks.
+ - Added dr_merge_arith_flags() as a convenience routine to merge arithmetic flags
+   for restoration done by outline code.
```

Comment:
s/outline/outlined/?  I don't think "outline code" is going to be familiar to many people; "outlined" makes it clearer IMHO.

Reply:
Done.